Backpack/src/main/res/values/themes.xml

```diff
@@ -35,4 +35,26 @@
     <item name="bpkTextXxxlHeavyAppearance">@style/bpkTextXxxlHeavySerif</item>
   </style>
 
+  <style name="BpkTheme.Relative">
```

Comment:
At first this was defined in the app theme, but this would require quite a lot of duplicated definitions when the only thing that needs to be outside of the library is the actual font definitions.